.github/workflows/shellcheck.yml

```diff
@@ -34,10 +34,6 @@ jobs:
             github.com:443
             pkg-containers.githubusercontent.com:443
       - uses: actions/checkout@v2
-      - name: Install shellcheck
-        run: brew install shellcheck
-        env:
-          HOMEBREW_NO_ANALYTICS: 1
```

Comment:
to be clear, i *do* want to always install the latest version of shellcheck; i don't want to be limited to whatever github deigns to ship.